src/main/java/org/gradle/profiler/jfr/JfrProfiler.java

```diff
@@ -45,4 +46,12 @@ protected JvmArgsCalculator jvmArgsWithInstrumentation(ScenarioSettings settings
     private File getJfrFile(ScenarioSettings settings) {
         return new File(settings.getScenario().getOutputDir(), settings.getScenario().getProfileName() + PROFILE_JFR_SUFFIX);
     }
+
+    @Override
+    public void validate(ScenarioSettings settings, Consumer<String> reporter) {
```

Comment:
This overwrites parent class' `validate()` https://github.com/gradle/gradle-profiler/blob/966f032d33f9a91847afe00084c62e69f58c85a1/src/main/java/org/gradle/profiler/InstrumentingProfiler.java#L86